apps/E2E/src/common/BasePage.ts

```diff
@@ -238,6 +242,49 @@ export abstract class BasePage {
     await FocusButton.addValue(scrollDownKeys);
   }
 
+  /* Scrolls to the specified or primary UI test element until it is displayed. */
+  async mobileScrollToTestElement(componentIdentifier?: string): Promise<void> {
```

Comment:
This has some code duplication from `scrollToTestElement()`.

Let's combine the two methods - this also aligns the specs better by calling the same function:

`scrollToTestElement(component?: WebDriverIO.Element)`

And then, we can simply use the `switch(this.platform)` to do the correct thing cross-plat

Reply:
Moving ahead as discussed offline with separate mobile scroll function since the parameter types are different and have limited code duplication. Also simplifies the functions and will be easier to diagnose misused function platform wise while writing tests.